bin/sirius-script-rabpm-mngr.py

```diff
@@ -96,30 +111,37 @@ def main():
 
     parser_reset = subparsers.add_parser("rffe-reset",
                                          help="Reset RFFE")
-    parser_reset.add_argument("-v", "--vslot", required=True,
-                              help="Virtual slot number to reset(calculated as "
+    parser_reset.add_argument("-v", "--vslots", nargs="+", type=int, default=list(range(11, 24)),
+                              help="Virtual slot numbers to reset(calculated as "
                               "(2*physical_slot-1) or (2*physical_slot))")
     add_rack_arg(parser_reset)
```

Comment:
I think it is interesting to have a `--all` for RFFE reset to (it seems this would be more frequently used, given the last occurrences).